docs/en/how_to_deploy_loomio/README.md

```diff
@@ -0,0 +1,271 @@
+
+Run this command to setup the Loomio database.
+
+```sh
+docker run --link postgres:db \
+           loomio/loomio:docker \
+           rake db:setup
+```
+
+the output should look like this:
+![database is setup](database_is_setup.png)
+
+## Starting Loomio
+
+note: link the SSL certificate, add an env for ssl
+
+Now start the Loomio app container:
+
+```sh
+docker run --name loomio \
+           --link postgres:db \
+           -v /loomio/uploads:/uploads/ \
+           -v /loomio/config:/config/ \
+           -p 80:3000 \
+           -d \
+           loomio/loomio:docker
+```
+
+Now you should be able visit https://yourhostname.com/ and see a login screen.
+
+If you run into trouble try the `docker logs` command. This returns the output of a daemonized docker container.
+
+```sh
+docker logs loomio
+```
+
+Other need to know docker commands include:
+* `docker ps` lists running containers.
+* `docker ps -a` lists all containers.
+* `docker stop <container_id or name>` will stop a container
+* `docker start <container_id or name>` will start a container
+* `docker restart <container_id or name>` will restart a container
+* `docker rm <container_id or name>` will delete a container
+* `docker pull loomio/loomio:latest` pulls the latest version of loomio
+* `docker help <command>` will help you understand commands
+
+To update Loomio to the latest image you'll need to stop, rm, pull, and run again.
+
+```sh
+docker stop loomio
+docker rm loomio
+docker pull loomio/loomio
+docker run
+```
+
+### Setup recurring jobs
+
+You need to install this crontab file so that Loomio will perform routine work such as emailing people when proposals are going to close.
+
+```sh
+echo "`docker exec loomio cat /loomio/docker/crontab`" >> /etc/crontab
+cat /etc/crontab
+```
+
+Looks like this:
+![Crontab installed](crontab_installed.png)
+
+## Setup inbound email
+
+One cool feature of Loomio is that people can reply to discussions from their email client.
+
+To do this, you need to create an MX DNS record to tell the internet what server is handling mail for your host.
+
+It's an MX record with priority 0 with the same hostname.
+
+![setup mx record](add_mx_record.png)
+
+The `loomio/mailin-docker` image runs [mailin](http://mailin.io) which is an SMTP server that converts emails to into web requests.
+
+```sh
+docker run --name mailin \
+           --link loomio:loomio \
+           -e WEBHOOK_URL=http://loomio:3000/email_processor/ \
+           -d loomio/mailin-docker
+```
+
+Note to self: link to a docker cheat sheet or write my own.
+
+## Setup live update
+
+
+
+```
+git clone https://github.com/letsencrypt/letsencrypt
+cd letsencrypt
+./letsencrypt-auto --help
+letsencrypt certonly --webroot -w /var/www/example -d example.com -d www.example.com -w /var/www/thing -d thing.is -d m.thing.is
+```
+
+need to progress to using docker-compose.yml
+
+Todo: How to setup an automatic build of your own Loomio fork to run your own set of plugins
```

Comment:
I can help with this. It's also the kind of thing we could pull out into the UI if there ends up being some interest in it.